test/common/http/header_map_impl_test.cc

```diff
@@ -372,15 +372,15 @@ class HeaderMapImplTest : public testing::TestWithParam<uint32_t> {
 public:
   HeaderMapImplTest() {
     // Set the lazy map threshold using the test parameter.
-    Runtime::LoaderSingleton::getExisting()->mergeValues(
+    scoped_runtime.mergeValues(
         {{"envoy.http.headermap.lazy_map_min_size", absl::StrCat(GetParam())}});
   }
 
   static std::string testParamsToString(const ::testing::TestParamInfo<uint32_t>& params) {
     return absl::StrCat(params.param);
   }
 
-  TestScopedRuntime runtime;
+  TestScopedRuntime scoped_runtime;
```

Comment:
nit: can we append an underscore for consistency?
```suggestion
  TestScopedRuntime scoped_runtime_;
```